tfe/resource_tfe_agent_pool.go

```diff
@@ -83,6 +105,13 @@ func resourceTFEAgentPoolRead(d *schema.ResourceData, meta interface{}) error {
 	// Update the config.
 	d.Set("name", agentPool.Name)
 	d.Set("organization", agentPool.Organization.Name)
+	d.Set("organization_scoped", agentPool.OrganizationScoped)
+
+	var wids []interface{}
```

Comment:
I am curious why did you decide to go with an slice of interface here?

Reply:
not needed now, moved `allowed_workspaces_ids` to another resource